docker/batch-single/README.md

```diff
@@ -79,7 +79,10 @@ $ chmod +x run.sh
 $ ./run.sh
 ```
 
+### Compute Environment AMI
```

Comment:
Do `WORKER_INIT_TIMEOUT`, `AWS_SNS_TOPIC_ARN`, and `DEBUG_TIMEOUT` need to be added to the variables section above?  They are in `run.sh.example`.  If not, let me know and I'll approve the PR.

Reply:
Good catch! I was actually missing a few others as well. I have updated the README.md to include all possible configurations.